content/src/cloud/features/remote-ui-enable.md

```diff
@@ -49,6 +53,15 @@ Activating remote control from outside your network only works if under [**Setti
 
 3. You can now connect to your Home Assistant using the **Remote address** URL.
 
+## Using an automation to enable and disable the remote connection
+
+The cloud component exposes two services to enable and disable the remote connection as part of an automation:
+
+- `cloud/remote_connect`
+- `cloud/remote_disconnect`
```

Comment:
These look more like events and not services.

Events are generally `<domain>/<event_type>`
Services are `<domain>.<service>`

`.` vs `/`

Reply:
Thank you for the catch! https://github.com/NabuCasa/support/pull/341